<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup_control_MainWindow</class>
 <widget class="QMainWindow" name="setup_control_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qudi: Setup Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="4">
       <widget class="QLabel" name="Read_A1Power_Label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="Set_A1Power_Button">
        <property name="text">
         <string>Set A1 Power</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="Read_A2Power_Button">
        <property name="text">
         <string>Read A2 Power</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="Repump_time_doubleSpinBox"/>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="Autofocus_Button">
        <property name="text">
         <string>Autofocus</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="Set_A2Power_DoubleSpinBox"/>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="StopAutoMeas_Button">
        <property name="text">
         <string>Stop Auto. Meas.</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="MW3_freq_DoubleSpinBox">
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="value">
         <double>70.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="Green_Button">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="MW2_power_DoubleSpinBox">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="MW1_power_DoubleSpinBox">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>MW2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>MW frequency (MHz)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="MW2_on_Button">
        <property name="text">
         <string>MW2 ON</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="A2_time_doubleSpinBox"/>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="Flipmirror_Button">
        <property name="text">
         <string>Flip Mirror</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="Set_A2Power_Button">
        <property name="text">
         <string>Set A2 Power</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>MW3</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="MW3_power_DoubleSpinBox">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QPushButton" name="SavePOIs_Button">
        <property name="text">
         <string>Save POIs</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="Read_A1Power_Button">
        <property name="text">
         <string>Read A1 Power</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="MW2_freq_DoubleSpinBox">
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="value">
         <double>70.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="Repump_Button">
        <property name="text">
         <string>Repump</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Lasers</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="StartAutoMeas_Button">
        <property name="text">
         <string>Start Auto. Meas.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Laser pulse time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="A1_time_doubleSpinBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="MW1_freq_DoubleSpinBox">
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="value">
         <double>70.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="A2_Button">
        <property name="text">
         <string>A2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="MW1_on_Button">
        <property name="text">
         <string>MW1 ON</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="A1_Button">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>A1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>MW Power (dBm)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QDoubleSpinBox" name="green_time_doubleSpinBox"/>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="Set_A1Power_DoubleSpinBox">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="MW3_on_Button">
        <property name="text">
         <string>MW3 ON</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>MW1</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QLabel" name="Read_A2Power_Label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>A1_Button</tabstop>
  <tabstop>A2_Button</tabstop>
  <tabstop>Repump_Button</tabstop>
  <tabstop>Green_Button</tabstop>
  <tabstop>A1_time_doubleSpinBox</tabstop>
  <tabstop>A2_time_doubleSpinBox</tabstop>
  <tabstop>Repump_time_doubleSpinBox</tabstop>
  <tabstop>green_time_doubleSpinBox</tabstop>
  <tabstop>MW1_freq_DoubleSpinBox</tabstop>
  <tabstop>MW1_power_DoubleSpinBox</tabstop>
  <tabstop>MW1_on_Button</tabstop>
  <tabstop>MW2_freq_DoubleSpinBox</tabstop>
  <tabstop>MW2_power_DoubleSpinBox</tabstop>
  <tabstop>MW2_on_Button</tabstop>
  <tabstop>MW3_freq_DoubleSpinBox</tabstop>
  <tabstop>MW3_power_DoubleSpinBox</tabstop>
  <tabstop>MW3_on_Button</tabstop>
  <tabstop>Set_A1Power_DoubleSpinBox</tabstop>
  <tabstop>Set_A1Power_Button</tabstop>
  <tabstop>Read_A1Power_Button</tabstop>
  <tabstop>Autofocus_Button</tabstop>
  <tabstop>Flipmirror_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
